lib/plugins/aws/package/compile/events/cloudFront/index.js

```diff
@@ -0,0 +1,336 @@
+'use strict';
+
+const _ = require('lodash');
+const url = require('url');
+const chalk = require('chalk');
+const ServerlessError = require('../../../../../../classes/Error');
+
+class AwsCompileCloudFrontEvents {
+  constructor(serverless, options) {
+    this.serverless = serverless;
+    this.options = options;
+    this.provider = this.serverless.getProvider('aws');
+    this.hooks = {
+      'package:initialize': this.validate.bind(this),
+      'before:package:compileFunctions': this.prepareFunctions.bind(this),
+      'package:compileEvents': this.compileCloudFrontEvents.bind(this),
+    };
+    this.logRemoveReminder.call(this);
+  }
+
+  logRemoveReminder() {
+    const funcKeys = this.serverless.service.getAllFunctions();
+    let isEventUsed = false;
+    if (funcKeys.length) {
+      isEventUsed = !!funcKeys.find(funcKey =>
```

Comment:
Good tip, forgot about that. Thanks :+1:

Will merge this now to avoid yet another round of reviews. Maybe we can refactor such usages in the future...